server/cache_test.go

```diff
@@ -291,13 +291,11 @@ func (s *testClusterInfoSuite) testStoreHeartbeat(c *C, cache *clusterInfo) {
 		c.Assert(cache.getStoreCount(), Equals, int(i+1))
 
 		stats := store.status
-		c.Assert(stats.LeaderCount, Equals, uint32(0))
 		c.Assert(stats.LastHeartbeatTS.IsZero(), IsTrue)
 
 		c.Assert(cache.handleStoreHeartbeat(storeStats), IsNil)
 
 		stats = cache.getStore(store.GetId()).status
-		c.Assert(stats.LeaderCount, Equals, uint32(1))
```

Comment:
Now it should be checked after region heartbeat, so I test it here: https://github.com/pingcap/pd/pull/575/files#diff-ab284d17e1bee930685c182610f4fa98R396